articles/ams-team-setups.md

```diff
@@ -0,0 +1,43 @@
+---
+authorName: Tijana Krstajic, Guido Angenendt
+authorGithubUsername: doogu
+issue: 91
+title: Operations Team Setups
+---
+# {{Operations team setups}}
+
+![Dummy Placeholder Image](./images/ams/AMS.png)
+ADM phases
+
+The Live phase of the application (the final phase in the ADM*), or better the moment in time when the actual users start using the application, is the point in the development lifecycle when the Customer gets a return on his investment. It is a live or die situation – the stability and the performance of the application in production can either guarantee your Customer’s success or otherwise lead to his failure. This is the main reason why considering as early as possible the collaboration between delivery and operations should be your point of attention already in the Discovery phase.
+In happy circumstances, the future members of the Operations team are involved first in the bid process (during Discovery phase) – like the Service Manager, the DevOps engineer as early as the Alpha phase (when the environments and CI/CD** get set up). Latest, these team members should get involved by the mid of Beta phase (or in other words Development or Delivery phase), in order to set up proper tools and processes to support Release Management, preventive maintenance as well as Incident Management.
+As the application approaches the Live phase, the team undergoes a shift in focus and setup. A Service Manager and a DevOps engineer get added to the team, and Developer and Quality Manager undergo a slight change of the mindset – their primary focus isn’t only delivering new features but also ensuring that the existing ones are performing in production and satisfying user needs.
```

Comment:
Is this description a good way of doing it? 
Or do you follow the pattern: Situation, Pain, Why, BetterWay.

To me the situation described sounds not very DevOpsy. 
Adding and removing people is not a good practice in my experience and not what DevOps means....